noaa/operations/default.json

```diff
@@ -0,0 +1,112 @@
+    {
+      "name": "index",
+      "operation-type": "index",
+      "bulk-size": 5000
+    },
+    {
+      "name": "range_query_range_field",
+      "operation-type": "search",
+      "body": {
+        "query": {
+          "range": {
+            "TRANGE": {
+              "gte": -10,
+              "lte": 10
+            }
+          }
+        }
+      }
+    },
+    {
+      "name": "range_query_range_field_in_conjunction_with_term_query",
+      "operation-type": "search",
+      "body": {
+        "query": {
+          "bool": {
+            "must": [
+              {
+                "term": {
+                  "station.keyword": "ASN00001018"
+                }
+              },
+              {
+                "range": {
+                  "TRANGE": {
+                    "gte": -10,
+                    "lte": 10
+                  }
+                }
+              }
+            ]
+          }
+        }
+      }
+    },
+    {
+      "name": "range_query_range_field_in_conjunction_with_terms_query",
+      "operation-type": "search",
+      "body": {
+        "query": {
+          "bool": {
+            "must": [
+              {
+                "terms": {
+                  "station.keyword": [
+                    "ASN00001018",
+                    "AQW00061705",
+                    "ARM00087453",
+                    "ARM00087576",
+                    "ASN00001006",
+                    "ASN00001007",
+                    "ASN00001010",
+                    "ASN00001013",
+                    "ASN00001018",
+                    "ASN00001019",
+                    "ARM00087121",
+                    "ASN00004032",
+                    "ASN00004046",
+                    "ASN00004019",
+                    "ASN00003105",
+                    "ASN00003100",
+                    "ASN00004083"
+                  ]
```

Comment:
> A weather station in this data set has at most 366 document which is 0,014% of the total amount of documents. So I think the 0.1 and 1% case is covered.

Arg, I made a mistake. A simple term query for weather station is 0,003%. The `terms` query matches with 5856 documents and that is 0,05%. So what I'll do is increase the number terms in the the `terms` query to get at least to 0,1%